src/containers/Signup/Signup.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+class Signup extends React.Component {
+    constructor (props) {
+        super(props)
+
+        this.state = {
+            inputs: {
+                username: "", 
+                email: "",
+                password: ""
+            }
+        }
+    }
```

Comment:
Fun fact: you can be a little more concise with setting an initial `state` 😉 

```js
class MyComponent extends React.Component {
    state = {
        someValue: true,
    };
}
```

This is functionally basically the same as manually writing out a `constructor` and setting `this.state`... just a little easier to write IMO.

Reply:
I actually prefer writing it this way, but since other team members wrote their class components with the constructor I was trying to follow the same style. Is one more preferable?

Reply:
🤷‍♂ not a hard preference either way. A couple points mildly in favor of the `state = ` way:

* It takes fewer lines, making your components easier to read & understand (less code to squint at)
* It discourages putting code in the `constructor`, which is rarely necessary

If folks use the `constructor` away nothing bad will happen. Totally up to you 😄